common/jvm/src/test/scala/com/redi/scalademo/business/NumericStringValidatorTest.scala

```diff
@@ -0,0 +1,38 @@
+package com.redi.scalademo.business
+
+import org.scalatest.{Matchers, path}
+
+class NumericStringValidatorTest extends path.FunSpec with Matchers {
```

Comment:
It sure does run on both JS and JVM under shared source set. Seemed unnecessary to run the same test twice, so I was back and forth on this. Business logic shouldn't fail based on your compilation platform.

At first I was surprise that the same test results were being printed twice, and then I realized what was happening...